crates/codegen/syntax_templates/src/shared/language.rs

```diff
@@ -14,41 +14,52 @@ pub use super::{
 const DEBUG_ERROR_MERGING: bool = false;
 
 impl ParseError {
-    pub(crate) fn new<T: Into<String>>(position: TextPosition, expected: T) -> Self {
+    #[allow(dead_code)]
+    pub(crate) fn new_at<T: Into<String>>(position: TextPosition, expected: T) -> Self {
```

Comment:
thoughts about `for_position()` and `for_range()` instead? to indicate that the first is for positions (vs "at").